zoo/services/assets/js/gitlab_envs.js

```diff
@@ -0,0 +1,110 @@
+const gitlabEnvs = {
```

Comment:
hmm... this seems really complex, do we need all this? we don't have to reuse the forms for the zoo environments, as the gitlab envs should be read only, so this should be as simple as getting the data from api and displaying some simple divs with the data

Reply:
Ok, i looks like it misunderstood US. Now it only show gitlab envs without using forms for zoo envs